templates/map.html.twig

```diff
@@ -1,6 +1,12 @@
-{% set url = mapboxEncode(conferenceCoordinates) %}
+{% if conferenceCoordinates|length == 1 %}
+    {% set url = mapboxEncode(conferenceCoordinates, 5) %}
+{% else %}
+    {% set url = mapboxEncode(conferenceCoordinates) %}
+{% endif %}
 
-{% if url == null %}
+{% if conferenceCoordinates|length == 0 %}
+    <div class="alert alert-warning">You don't have any conference to show on the map. This may be because they are all online.</div>
```

Comment:
I really don't like the "alert" thing. We use "warning" or "danger" when the user does something wrong. here the user did not do anything wrong.

If you really want to put something, use a muted text